CONTRIBUTING.rst

```diff
@@ -328,6 +350,43 @@ Milestone size:
 * If a milestone contains too much, move some to the next milestone.
 * Err on the side of more frequent patch releases.
 
+Process: Pull Request merging and HISTORY.rst maintenance
+---------------------------------------------------------
+
+If you merge a pull request, you're responsible for updating `AUTHORS.rst` and `HISTORY.rst`
+
+When you're processing the first change after a release, create boilerplate following the existing pattern:
```

Comment:
I'm not too sure if this belongs in here. It is already covered to some degree in the https://github.com/audreyr/cookiecutter/blob/master/docs/core_committer_guide.rst